### PR TITLE
add depenedency note to docs #20732

### PR DESCRIPTION
**Pull request by note**
Resolves https://github.com/akka/akka/issues/20732. 

@ktoso: please let me know if it's what you meant (in ticket you reffered to `akka-testkit` but I guess you meant `akka-http-testkit`)

### CODE REVIEW

**Comment by drewhk** [comment]
LGTM, thanks!

File: akka-docs/rst/scala/http/routing-dsl/testkit.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Route TestKit
 One of Akka HTTP's design goals is good testability of the created services.
 For services built with the Routing DSL Akka HTTP provides a dedicated testkit that makes efficient testing of
 route logic easy and convenient. This "route test DSL" is made available with the *akka-http-testkit* module.
+To use it include the following dependency::
+
+  "com.typesafe.akka" %% "akka-http-testkit" % "2.4-SNAPSHOT"
```
Comment (johanandren) [inline akka-docs/rst/scala/http/routing-dsl/testkit.rst]: Use `@version@` instead (it is then automagically replaced by the specific akka version for every release).

**Comment by johanandren** [comment]
OK TO TEST

**Comment by ktoso** [comment]
LGTM, yep exactly. Thanks a lot!

**Pull request merged**